test/xrt/23_ctrlpkt_config/run.lit

```diff
@@ -0,0 +1,12 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// REQUIRES: ryzen_ai
+// RUN: %python %S/aie.py
+// RUN: %python aiecc.py --no-aiesim --aie-generate-cdo --no-compile-host --generate-ctrl-pkt-overlay --xclbin-name=base.xclbin %S/base.mlir
+// RUN: %python aiecc.py --no-aiesim --aie-generate-ctrlpkt --aie-generate-npu --no-compile-host --generate-ctrl-pkt-overlay --npu-insts-name=aie_run_seq.txt aie.mlir
+// RUN: aie-translate -aie-ctrlpkt-to-bin -aie-sequence-name=configure aie.mlir.prj/ctrlpkt.mlir -o ctrlpkt.txt
+// RUN: aie-opt -aie-ctrl-packet-to-dma -aie-dma-to-npu aie.mlir.prj/ctrlpkt.mlir -o ctrlpkt_dma_seq.mlir
+// RUN: aie-translate -aie-npu-instgen -aie-sequence-name=configure ctrlpkt_dma_seq.mlir -o ctrlpkt_dma_seq.txt
+// RUN: g++-13 %S/test.cpp -o test.exe -std=c++23 -Wall %xrt_flags -lrt -lstdc++ -lboost_program_options -lboost_filesystem
+// RUN: %run_on_npu ./test.exe -x base.xclbin -k MLIR_AIE -i aie_run_seq.txt -c ctrlpkt_dma_seq.txt -p ctrlpkt.txt -v 2
```

Comment:
What mlir-air feature is this testing? It seems like an mlir-aie test to me.

Reply:
The air feature tested here is basically the `use-pkt-flow-at-shim-dma=true` flag in `air-to-aie` pass (see aie.py), whose purpose is to enable control packet reconfiguration from shim.

This test could serve as an integration test, where I wanted to make sure that any air code developments in the future remain configurable via control packets.

Reply:
> The air feature tested here is basically the `use-pkt-flow-at-shim-dma=true` flag in `air-to-aie` pass (see aie.py), whose purpose is to enable control packet reconfiguration from shim.

ah, I missed that detail. Thanks!

> This test could serve as an integration test, where I wanted to make sure that any air code developments in the future remain configurable via control packets.